### PR TITLE
configured pre-commit

### PR DESCRIPTION
**Pull request by jcsekar**
pre-commit would help the developers to catch code style related errors during commit layer instead of waiting for CI.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,7 +15,8 @@ def get_requirements():
         'requests==2.22.0',
         'PyJWT==1.7.1',
         'python-dotenv==0.10.3',
-        'simple-chalk==0.1.0'
+        'simple-chalk==0.1.0',
+        'pre-commit'
```
Comment (Haider8) [inline setup.py]: No need to add it here

File: requirements.txt [hunk 2]
```diff
@@ -6,3 +6,4 @@ PyJWT==1.7.1
 python-dotenv==0.10.3
 pylint==2.4.2
 simple-chalk==0.1.0
+pre-commit
```
Comment (Haider8) [inline requirements.txt]: Write the version also

**Comment by Haider8** [comment]
@jcsekar Have you test this PR correctly?

**Comment by jcsekar** [comment]
> @jcsekar Have you test this PR correctly?

Yes tested it.

File: requirements.txt (hunk 2 above)
Comment (jcsekar) [reply requirements.txt]: done

File: setup.py (hunk 1 above)
Comment (jcsekar) [reply setup.py]: removed

**Comment by Haider8** [comment]
@jcsekar Thank you for sending this PR, be sure to check out some other issues also :)

:1st_place_medal: If you like this project, please give this repository a star to show us your support :1st_place_medal:

**Pull request merged**